package.json

```diff
@@ -17,7 +17,10 @@
     "publish:beta": "lerna version prerelease --preid beta --force-publish --yes && release --pre",
     "test": "jest",
     "test:watch": "cross-env NODE_ENV=test jest --watchAll",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "scaffold": "npm run lerna:bootstrap && npm run lerna:link",
+    "lerna:link": "lerna link",
+    "lerna:bootstrap": "lerna bootstrap"
```

Comment:
Could you put this in the prettier PR instead?

Reply:
Ah, but its already closed 😅

Reply:
Yeh no worries, I can help with that. It should be separate from both PRs anyway